packages/auto-consensus/src/balances.ts

```diff
@@ -2,6 +2,29 @@ import { activate, Api } from '@autonomys/auto-utils'
 import { account } from './account'
 import type { BalanceData } from './types/balance'
 
+/**
+ * Retrieves the total token issuance of the network.
+ * 
+ * This function queries the blockchain to get the total amount of tokens that have been issued
+ * across the entire network. This includes all tokens in circulation, staked, reserved, and frozen.
+ * 
+ * @param networkId - Optional network identifier. If not provided, uses the default network
+ * @returns Promise that resolves to a Codec containing the total issuance value as a hex string
+ * @throws Error if the network connection fails or the query is unsuccessful
+ * 
+ * @example
+ * ```typescript
+ * import { totalIssuance } from '@autonomys/auto-consensus'
+ * 
+ * // Query total issuance for default network
+ * const issuance = await totalIssuance()
+ * const totalTokens = BigInt(issuance.toString())
+ * console.log(`Total tokens issued: ${totalTokens}`)
+ * 
+ * // Query for specific network
+ * const geminiIssuance = await totalIssuance('gemini-3h')
```

Comment:
Do not use specific network in names, just use something like `networkIssuance`.